src/Datadog.Trace/HttpOverStreams/DatadogHttpClient.cs

```diff
@@ -21,7 +22,7 @@ internal class DatadogHttpClient
         public async Task<HttpResponse> SendAsync(HttpRequest request, Stream requestStream, Stream responseStream)
         {
             await SendRequestAsync(request, requestStream).ConfigureAwait(false);
-            return ReadResponse(responseStream);
+            return await ReadResponseAsync(responseStream);
```

Comment:
Shouldn't this `await` have `ConfigureAwait(false)`? Alternatively, can we return the task from `ReadResponseAsync()` without awaiting it?

Reply:
Yeah, you're right, no need to await

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1218/commits/1db429214309b593581ad47b1bffbe8b67d599ce